uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Form/AbsencePeriod.php

```diff
@@ -55,7 +55,7 @@ public function buildQuickForm() {
     $this->addButtons($this->getAvailableButtons());
 
     CRM_Core_Resources::singleton()->addStyleFile('uk.co.compucorp.civicrm.hrleaveandabsences', 'css/leaveandabsence.css');
-    CRM_Core_Resources::singleton()->addScriptFile('uk.co.compucorp.civicrm.hrleaveandabsences', 'js/crm/hrleaveandabsences.form.absenceperiod.js');
+    CRM_Core_Resources::singleton()->addScriptFile('uk.co.compucorp.civicrm.hrleaveandabsences', 'js/dist/crm-form-absenceperiod.min.js', 1001);
```

Comment:
Any reason why the weight is hardcoded here?

Reply:
Yes, needs to appear after reqangular is injected.

Reply:
Can we add a comment as to why this is so. A bit difficult to get that by merely looking at the code.

Reply:
Can we define constants like this `CRM_Core_Resources::DEFAULT_WEIGHT`, but for example `CRM_Core_Resources::REQANGULAR` and `CRM_Core_Resources::AFTER_REQANGULAR`?

Reply:
That would be nice!

Reply:
Done.